PennMobile/src/main/java/com/pennapps/labs/pennmobile/DiningInfoFragment.java

```diff
@@ -174,11 +162,31 @@ public void fillInfo(){
         return vertical;
     }
     @Override
+    public void onResume() {
+        super.onResume();
+        mActivity.getActionBarToggle().setDrawerIndicatorEnabled(false);
+        mActivity.getActionBarToggle().syncState();
+        getActivity().setTitle(mDiningHall.getName());
+        if (map == null) {
+            map = mapFragment.getMap();
+            if (map != null) {
```

Comment:
Yes, it can be null, and wait for it...deprecated
![screenshot - 03222016 - 05 40 33 am](https://cloud.githubusercontent.com/assets/929309/13947393/a86df1ee-eff0-11e5-918e-1678bb3f9360.png)